docs/content/latest/troubleshoot/cluster/replace_failed_peers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Fix 2/3 failed peers
```

Comment:
Perhaps the title should be "Manual remote bootstrap when two of three peers fail" @bogdan?

Reply:
Actually, I think given @ddorian 's comment about RF5 as well, it would make sense to change this to maybe refer to losing a majority, but then when we start discussing the example, we explicitly say we'll use RF3 as an example.

Reply:
Changed to `Manual remote bootstrap when a majority of peers fail`